README.rst

```diff
@@ -107,7 +107,8 @@ how to create a bucket.
     blob = bucket.get_blob('/remote/path/to/file.txt')
     print blob.download_as_string()
     blob.upload_from_string('New contents!')
-    bucket.upload_file('/remote/path/storage.txt', '/local/path.txt')
+    blob2 = storage.Blob('/remote/path/storage.txt', bucket)
```

Comment:
@jgeewax I forgot to implement the factories when moving to the client pattern. I was actually working on it at this very moment.